src/core/Form/Select/MultiSelect/MultiSelect/MultiSelect.md

```diff
@@ -148,3 +148,30 @@ const animals = [
     <button onClick={() => setSelectedAnimals([{ labelText: 'Snail', uniqueItemId: 'snail-321' }])}>Snail</button>
 </>
 ```
+
+### Disabled
+
+```
+const foods = [
+  {
+    labelText: 'Pizza',
+    uniqueItemId: 'pizza-123'
+  },
+  {
+    labelText: 'Burger',
+    uniqueItemId: 'burger-321'
+  }
+];
+
+<>
+    <MultiSelect
+      disabled={true}
+      items={foods}
+      labelText="Food"
+      visualPlaceholder="Selection disabled"
+      ariaSelectedAmountText="foods selected"
```

Comment:
I think this should be more generic, because it should probably be uniform across the service using MultiSelect despite the context of the individual MultiSelect. So probably just 'items selected'.

Reply:
Done!